layouts/partials/footer.html

```diff
@@ -9,6 +9,10 @@
 {{ template "_internal/google_analytics_async.html" . }}
 {{ end }}
 
+{{- if or (eq .Site.Params.mode "auto") (eq .Site.Params.mode "light") -}}
+  <script src="{{ .Site.BaseURL }}js/toggle.dark.js"></script>
```

Comment:
@athul maybe need to if else logic here. So if dark mode is there, then load toggle.light.js